Sources/SWBCore/MacroConfigFileLoader.swift

```diff
@@ -242,7 +242,7 @@ final class MacroConfigFileLoader: Sendable {
                 return MacroConfigFileParser(byteString: data, path: path, delegate: delegate)
             }
 
-            mutating func foundMacroValueAssignment(_ macroName: String, conditions: [(param: String, pattern: String)], value: String, path: Path, line: Int, startColumn: Int, endColumn: Int, parser: MacroConfigFileParser) {
+            mutating func foundMacroValueAssignment(_ macroName: String, conditions: [(param: String, pattern: String)], value: String, path: Path, startLine: Int, endLine: Int, startColumn: Int, endColumn: Int, parser: MacroConfigFileParser) {
```

Comment:
Could `startLine: Int, endLine: Int, startColumn: Int, endColumn: Int` be something like `rhs: Range<SourceLocation>`?

Reply:
I guess yah, maybe in a future PR :)